internal/provider/resource_ably_queue_test.go

```diff
@@ -0,0 +1,83 @@
+package ably_control
+
+import (
+	"fmt"
+	"testing"
+
+	ably_control_go "github.com/ably/ably-control-go"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccAblyQueue(t *testing.T) {
+	app_name := acctest.RandStringFromCharSet(15, acctest.CharSetAlphaNum)
+	queue_name := acctest.RandStringFromCharSet(15, acctest.CharSetAlphaNum)
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAblyQueueConfig(app_name, ably_control_go.NewQueue{
+					Name:      queue_name,
+					Ttl:       44,
+					MaxLength: 83,
+					Region:    ably_control_go.EuWest1A,
+				}),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					resource.TestCheckResourceAttr("ably_app.app0", "name", app_name),
+					resource.TestCheckResourceAttr("ably_queue.queue0", "name", queue_name),
+					resource.TestCheckResourceAttr("ably_queue.queue0", "ttl", "44"),
+					resource.TestCheckResourceAttr("ably_queue.queue0", "max_length", "83"),
+				),
+			},
+			{
+
+				Config: testAccAblyQueueConfig(app_name, ably_control_go.NewQueue{
+					Name:      queue_name + "new",
+					Ttl:       99,
+					MaxLength: 83,
+					Region:    ably_control_go.UsEast1A,
+				}),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					resource.TestCheckResourceAttr("ably_app.app0", "name", app_name),
+					resource.TestCheckResourceAttr("ably_queue.queue0", "name", queue_name+"new"),
+					resource.TestCheckResourceAttr("ably_queue.queue0", "ttl", "99"),
```

Comment:
```suggestion
					resource.TestCheckResourceAttr("ably_queue.queue0", "ttl", "30"),
```

Reply:
I changed the number at some point, Maybe the change wasn't commited.